pom.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-spring-boot-parent</artifactId>
+		<version>1.2.0</version>
+	</parent>
+	<artifactId>item-handler</artifactId>
+	<version>unversioned</version>
+	<name>item-handler</name>
+	<description>Handler service dealing with placed orders and producing individual items to be processed internally</description>
+
+	<properties>
+		<java.version>1.8</java.version>
+		<org.mapstruct.version>1.3.0.Final</org.mapstruct.version>
+		<jacoco-maven-plugin.version>0.7.7.201606060606</jacoco-maven-plugin.version>
+		<junit-jupiter-engine-version>5.2.0</junit-jupiter-engine-version>
+		<mockito-junit-jupiter-version>2.18.0</mockito-junit-jupiter-version>
+		<hamcrest-all-version>1.3</hamcrest-all-version>
+		<start-class>uk.gov.companieshouse.itemhandler.ItemHandlerApplication</start-class>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-rest</artifactId>
+		</dependency>
```

Comment:
Is `item-handler` going to be accessing any databases?

Reply:
I've updated this to use the web starter.